tests/unit/lib/utils/test_name_utils.py

```diff
@@ -0,0 +1,162 @@
+"""
+Tests for samcli.lib.utils.name_utils module
+"""
+
+import pytest
+
+from samcli.lib.utils.name_utils import normalize_lambda_function_name, InvalidFunctionNameException
+
+
+class TestNormalizeLambdaFunctionName:
```

Comment:
Please use format where the test class extends `TestCase` (and assertions are `self.assert*(`) like other files: https://github.com/bchampp/aws-sam-cli/blob/develop/tests/unit/lib/utils/test_architecture.py#L18

Reply:
Sounds good. Addressed.